### DISCUSSION TITLE
Error while using DuckDNS plugin

### DISCUSSION THREAD

**Discussion by sashanknjs** [infinityofspace/certbot_dns_duckdns#147] [General]
```
(sage-env) sage@sage:~/sage $ sudo certbot certonly \
  --non-interactive \
  --agree-tos \
  --email aitechies9@gmail.com \
  --preferred-challenges dns \
  --authenticator dns-duckdns \
  --dns-duckdns-credentials </home/sage/sage/duckdns.ini> \
  --dns-duckdns-propagation-seconds 60 \
  -d "sageai.duckdns.org"

```
I get the following error when I try to run the above code

usage: 
  certbot [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
certificate. 
certbot: error: unrecognized arguments: --dns-duckdns-credentials 60

**Comment by infinityofspace** [comment]
Did you used the path `</home/sage/sage/duckdns.ini>` as argument or is this a anonymization of the real path?

The example
```bash
--dns-duckdns-credentials </path/to/your/duckdns.ini>
```
from the Readme have to be adjusted such that the real path replaces the whole substring `</path/to/your/duckdns.ini>` with the correct path. In your case it would be:

```bash
 --dns-duckdns-credentials /home/sage/sage/duckdns.ini
```

If this doesn't fix the problem, can you provide some more information about your and system:
1. What are the outputs of the commands? `certbot plugins` and `sudo certbot plugins`
2. How do you installed the plugin and certbot?
3. What version of `certbot_dns_duckdns` and `certbot` are installed? You can use the commands `pip show certbot_dns_duckdns` and `certbot --version`
4. What OS are you using?
